packages/dropdowns/src/elements/Menu/Items/HeaderIcon.spec.tsx

```diff
@@ -0,0 +1,83 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { render } from 'garden-test-utils';
+import { Dropdown, Trigger, Menu, HeaderItem, HeaderIcon } from '../../..';
+import { DEFAULT_THEME } from '@zendeskgarden/react-theming';
+
+const ExampleSvg = React.forwardRef<SVGSVGElement, {}>((props, ref) => (
+  <svg ref={ref} {...props} />
+));
+
+describe('HeaderIcon', () => {
+  it('passes ref to underlying DOM element', () => {
+    const ref = React.createRef<HTMLDivElement>();
+
+    const { getByTestId } = render(
+      <Dropdown>
+        <Trigger>
+          <button data-test-id="trigger">Test</button>
```

Comment:
I noticed that this `data-test-id` isn't used in a couple of tests and can probably be removed.

Reply:
Removed.